src/store/plugins/pendingTransactionHandler.js

```diff
@@ -1,36 +1,15 @@
-import { useModals } from '../../composables';
-import { MODAL_SPEND_SUCCESS, watchUntilTruthy } from '../../popup/utils';
+import { watchUntilTruthy } from '../../popup/utils';
 
 export default async (store) => {
-  const { openModal } = useModals();
-
   const waitTransactionMined = async ({
-    hash, type, tipUrl, recipient: recipientId, tx, ...otherTx
+    hash,
+    type,
   }) => {
     const network = store.getters.activeNetwork?.networkId;
-    try {
-      const transaction = await store.getters['sdkPlugin/sdk'].poll(hash);
-      const showSpendModal = () => openModal(MODAL_SPEND_SUCCESS, {
-        transaction: {
-          tipUrl,
-          ...otherTx,
-          ...transaction,
-          ...(type === 'spendToken')
-            ? { tx: { ...transaction.tx, recipientId, amount: tx.amount } }
-            : {},
-        },
-      });
 
-      switch (type) {
-        case 'tip':
-        case 'spend':
-          showSpendModal();
-          break;
-        case 'spendToken':
-          store.dispatch('fungibleTokens/loadTokenBalances');
-          showSpendModal();
-          break;
-        default:
+    try {
+      if (type === 'spendToken') {
+        store.dispatch('fungibleTokens/loadTokenBalances');
```

Comment:
I think we can remove this, since the balance of fungible token is not updating as fast as transaction mining, and we are currently fetching the fungible token balance every 10 seconds.